include/swift/AST/DiagnosticEngine.h

```diff
@@ -826,7 +837,8 @@ namespace swift {
     void emitTentativeDiagnostics();
 
   public:
-    static const char *diagnosticStringFor(const DiagID id);
+    static const char *diagnosticStringFor(const DiagID id,
+                                         bool printDiagnosticName);
```

Comment:
Nitpick: Needs some extra whitespace

Reply:
Huh, clang-format seems to be insisting on the weird formatting here. Anyway, fixed